.eslintrc

```diff
@@ -3,6 +3,6 @@
     "eslint:recommended", "plugin:@typescript-eslint/recommended",
   ],
   "parser": "@typescript-eslint/parser",
-  "plugins": "@typescript-eslint",
+  "plugins": ["@typescript-eslint"],
```

Comment:
This was the config mistake you talked about?

Reply:
no it was a missing tsconfig prop. more specifically `esModuleInterop` had to be true in order to support default imports as opposed to `import * as something`